source/features/clean-repo-tabs.tsx

```diff
@@ -32,12 +32,13 @@ function setTabCounter(tab: HTMLElement, count: number): void {
 }
 
 function onlyShowInDropdown(id: string): void {
-	const tabItem = $optional(`[data-tab-item$="${id}"]`);
-	if (!tabItem && pageDetect.isEnterprise()) { // GHE #3962
+	// TODO: Use selector observer
+	const tabItem = $optional(`li:not([hidden]) > [data-tab-item$="${id}"]`);
+	if (!tabItem) { // #3962 #7140
```

Comment:
I didn't want to mess too much with this feature, but this could be rewritten as:

```js
observe('both data-tab-items', onlyShowInDropdown)
```

and then extract the string ID inside `onlyShowInDropdown`, maybe the whole `dataset.tabItem` can be used